src/main/java/duke/Over.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+/**
+ * Encapsulates boolean value to be used like a mutable object.
+ */
+public class Over {
+    boolean isOver;
```

Comment:
Should this be made public? 
From [SE-Guide](url): Class variables should never be declared public unless the class is a data class with no behavior. This rule does not apply to constants.